onnxruntime/python/tools/kernel_explorer/kernels/kernel_explorer.py

```diff
@@ -60,3 +60,86 @@
 
 # pylint: disable=wrong-import-position, disable=unused-import, disable=wildcard-import
 from _kernel_explorer import *  # noqa
+
+"""Benchmark Reporter"""
+
+from abc import abstractmethod
+from contextlib import contextmanager
+from dataclasses import dataclass
+
+
+@dataclass
+class InstanceStatus:
+    name: str
+    dtype: str
+    milliseconds_duration: float
+
+    def __lt__(self, other):
+        if "Tunable" in self.name or other.duration < 0:
+            return True
+        if "Tunable" in other.name or self.duration < 0:
+            return False
+
+        return self.duration < other.duration
+
+    @property
+    def duration(self):
+        return self.milliseconds_duration * 1000
+
+    @abstractmethod
+    def report(self) -> str:
+        ...
```

Comment:
> ...

Consider replacing `...` with `raise NotImplementedError()` here. Otherwise you may have a silent failure if a subclass forgets to implement this method.

Reply:
Done.